docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1348 @@
+{
```

Comment:
<p>Does it mean that we lost the auto-hide functionality?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Yes but I think <code>@#title</code>  should be kept for each cell in the plotting section actually. This way 'auto-hide' will be enabled when you click "run on colab" I believe. I'm not sure how it will appear on the website, though.</p>

Reply:
<p>Re-enabled auto-hide. </p>